conftest.py

```diff
@@ -237,3 +239,54 @@ def remote_file(request):
 
     session.delete(new_connection)
     session.flush()
+
+
+@pytest.fixture
+def database_table_fixture(request):
```

Comment:
Would be good to add a. single comment on what this is for and when we should use this fixture